csrc/mmdeploy/device/cpu/cpu_device.cpp

```diff
@@ -11,13 +11,16 @@ class CpuHostMemory : public NonCopyable {
  public:
   CpuHostMemory() : size_(), data_(), owned_data_{false} {}
   Result<void> Init(size_t size, size_t alignment) {
-    if (alignment != 1) {
-      return Status(eNotSupported);
-    }
-    data_ = std::malloc(size);
+    size_t space = size + alignment - 1;
```

Comment:
`space = (size + alignment - 1)  &  ~(alignment - 1)`

If mmdeploy depends on c++17, use `std::aligned_alloc`  https://en.cppreference.com/w/cpp/memory/c/aligned_alloc

Reply:
https://github.com/open-mmlab/mmdeploy/actions/runs/3407199937/jobs/5666603714#step:5:69
https://en.cppreference.com/w/cpp/algorithm/exclusive_scan
SDK build failed with exclusive_scan, I guess SDK is build with c++14.